homework2/2.1/2.1/StackArray.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace _2._1
+{
+    public class StackArray : IStack
+    {
+        private double[] stackElements;
+        private int size;
+        public StackArray()
+        {
+            stackElements = new double[10];
+            size = 0;
+        }
+        public void Push(double value)
```

Comment:
```suggestion
        }
        
        public void Push(double value)
```
Методы всегда разделяются пустой строкой